src/main/c/Makefile

```diff
@@ -104,7 +115,14 @@ libsimdpp-2.0-rc2.tar.gz:
 libsimdpp-2.0-rc2: libsimdpp-2.0-rc2.tar.gz
 	tar -xzf libsimdpp-2.0-rc2.tar.gz
 
-$(shared_library): libsimdpp-2.0-rc2 ${OBJECTS}
-	mkdir -p $(dir $(shared_library))
-	${CXX} ${LIBFLAGS} ${LIBDIRS} ${CXXFLAGS} ${OBJECTS} -o $@ 
+ifneq ($(boot_library),)
+$(boot_library): NativeBoot.o
+	-mkdir -p $(basename $(boot_library))
```

Comment:
why the `-`, doesn't `-p` ensure we don't get an error if this directories already exist? And if there is some other issue preventing creation, this should fail?

Reply:
Probably.  But it's customary to use the "-" on an idempotent do-this-if-necessary
command.